msg/VarResultCodes.msg

```diff
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: 2022-2023, Yaskawa America, Inc.
```

Comment:
This should probably be YEU instead of YAI. Also, the year can be bumped up.

Reply:
You are right. I forgot about that. This need to be changed in the services as well.